reporter/src/main/kotlin/reporters/evaluatedmodel/EvaluatedModelMapper.kt

```diff
@@ -306,6 +307,11 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
             detectedLicenses = detectedLicenses,
             detectedExcludedLicenses = detectedExcludedLicenses,
             concludedLicense = pkg.concludedLicense,
+            effectiveLicense = input.licenseInfoResolver.resolveLicenseInfo(pkg.id).filterExcluded().effectiveLicense(
+                LicenseView.CONCLUDED_OR_DECLARED_AND_DETECTED,
```

Comment:
Have license choices been already applied and if so, does the view need to be consistent with what was supplied to `applyChoices()` as view?

Reply:
Yes, the choices are applied inside the `effectiveLicense` function based on the supplied view.